src/activities/activity.types.ts

```diff
@@ -92,293 +176,791 @@ export enum ActivityTypes {
   'cycling_gravel_cycling' = 'Cycling',
   'Biking' = 'Cycling',
   'biking' = 'Cycling',
-  'e_biking' = 'E Biking',
-  'E Biking' = 'E Biking',
-  'EBikeRide' = 'E Biking',
   'Ride' = 'Cycling',
-  'cycling_mountain' = 'Mountain biking',
-  'MountainBiking' = 'Mountain biking',
-  'Mountain Biking' = 'Mountain biking',
-  'cycling_cyclocross' = 'Mountain biking',
-  'mountain' = 'Mountain biking',
-  'Mountain biking' = 'Mountain biking',
+  /**
+   * Indoor Cycling
+   */
+  'cycling_indoor_cycling' = 'Indoor Cycling',
```

Comment:
An example perhaps of what you mean?

Reply:
With something like this https://regex101.com/r/juKll7/1/ to catch cycling activities.
```typescript
map = [
...
{pattern: /^(cycling|bike|ride|...)$/gmi, type: Activity.Cycling}
...
]
```

I think it's more difficult than i thought :/ Forget it.

Reply:
I think to let it fail and investigate why / who adds a "strange" type is better imo. 

you can if you want fallback to something and issue a log (wanting at your side and to the user). The case is also that sometimes even, users like to edit GPX etc. Just saying. 

That said I think the important think in this PR to agree up is the produced value on the enum. 
Because I suppose that is what you will be storing, stats will use etc.... Probably also persisted in DB. 

So: `Backcountry Skiing` vs `Backcountry skiing` ? 

Or something else?

Reply:
`Backcountry Skiing`, in my opinion. But choose format you like ;)

FYI I will convert your sports to Strava ones in the Elevate Desktop app (because of legacy code working on Strava types). Others sports (not Strava ones) will be sports-libs types.